Gemfile

```diff
@@ -49,6 +49,9 @@ gem 'jwt', '~> 2.2.2'
 gem 'graphql', '~> 1.13.0'
 gem 'graphql-batch'
 
+# A bundled gem since Ruby 3.0
+gem 'rss' if RUBY_VERSION >= '3.0'
```

Comment:
Question is if we should make this conditional or also pull it in on Ruby 2.7.

In this current form the dependency generator for RPM will use the Ruby version that runs it (which for me is Ruby 3.1 since I'm on Fedora, but could be different for others). On the other hand, Debian packaging will use the real Ruby.

Reply:
it would be nice to avoid the if condition here IMHO, but not a big deal if that's not possible, it's just something we'd need to drop eventually when we support Ruby 3+

Reply:
> it would be nice to avoid the if condition here IMHO

Just curious, how else would you do it?